src/js/components/Drop/stories/Lazy.js

```diff
@@ -15,122 +15,113 @@ const lazyTheme = deepMerge(grommet, {
 
 const finalLazyPad = 'xlarge';
 
-class LazyDrop extends Component {
-  state = { pad: 'small' };
+const LazyDrop = () => {
+  const [pad, setPad] = useState('small');
 
-  topLeftTargetRef = createRef();
+  const topLeftTargetRef = useRef(null);
 
-  topRightTargetRef = createRef();
+  const topRightTargetRef = useRef(null);
 
-  bottomLeftTargetRef = createRef();
+  const bottomLeftTargetRef = useRef(null);
 
-  bottomRightTargetRef = createRef();
+  const bottomRightTargetRef = useRef(null);
 
-  componentDidMount() {
-    this.forceUpdate();
-    setTimeout(() => this.setState({ pad: finalLazyPad }), 2000);
-  }
+  const [, forceUpdate] = useReducer(x => x + 1, 0);
 
-  render() {
-    const { pad } = this.state;
-    return (
-      <Grommet theme={lazyTheme} full>
-        <Box fill justify="between" pad="large" gap="small">
-          <Box direction="row" justify="between" pad={{ horizontal: 'small' }}>
-            <Box
-              background="dark-3"
-              pad="medium"
-              align="center"
-              ref={this.topLeftTargetRef}
+  useEffect(() => {
+    forceUpdate();
+    setTimeout(() => setPad({ pad: finalLazyPad }), 2000);
+  }, []);
+
+  return (
+    <Grommet theme={lazyTheme} full>
+      <Box fill justify="between" pad="large" gap="small">
+        <Box direction="row" justify="between" pad={{ horizontal: 'small' }}>
+          <Box
+            background="dark-3"
+            pad="medium"
+            align="center"
+            ref={topLeftTargetRef}
+          >
+            Target
+          </Box>
+          {topLeftTargetRef.current && (
+            <Drop
+              align={{ top: 'bottom', left: 'left' }}
+              target={topLeftTargetRef.current}
+              responsive
             >
-              Target
-            </Box>
-            {this.topLeftTargetRef.current && (
-              <Drop
-                align={{ top: 'bottom', left: 'left' }}
-                target={this.topLeftTargetRef.current}
-                responsive
+              <Box
+                height={pad === 'small' ? 'xsmall' : undefined}
```

Comment:
This logic for the box `height` appears in 3 different places as the same code. Wouldn't be good store this in a `const`?

Reply:
Sounds good, thanks.